docs/micro-lc/overview.md

```diff
@@ -4,17 +4,17 @@ title: Micro Launch Complex
 sidebar_label: Overview
 ---
 
-`micro-lc` is the open source solution for building flexible, multi-tenant front-end applications following the
+`micro-lc` is the open source solution for building flexible, multi-tenant frontend applications following the
```

Comment:
TBH seems that the right form is `front end`
https://english.stackexchange.com/questions/34447/is-it-front-end-frontend-or-front-end